zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, ParagraphRuntimeInfo> runtimeInfos;
```

Comment:
Yes we dont need to persist `runtimeInfos`.
If we add `transient` modifier, the `runtimeInfos` will not be serialized and sent to client browser.
Any suggestions to solve this? @Leemoonsoo